lib/horizon/trade_aggregation.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Stellar.Horizon.TradeAggregation do
+  @moduledoc """
+  Represents a `TradeAggregation` resource from Horizon API.
+  """
+
+  @behaviour Stellar.Horizon.Resource
+
+  alias Stellar.Horizon.Mapping
+
+  @type t :: %__MODULE__{
+          timestamp: non_neg_integer(),
+          trade_count: non_neg_integer(),
+          base_volume: float(),
+          counter_volume: float(),
+          avg: float(),
+          high: float(),
+          high_r: map(),
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏We could create a `@type rational_mapping` and use it for `high_r`, `open_r`, `close_r` fields in order to parse `n` and `d` to an integer type because I noticed that `n` and `d` currently are strings.
Similar to this approach:
https://github.com/kommitters/stellar_sdk/blob/ab5207eaaf21237ac23d9c125cbaa42cd37e3e0f/lib/horizon/asset.ex#L38-L50 

What do you think?

Reply:
Yes, I agree, we can map these fields!!!